apps/web/src/components/docs/docs.const.ts

```diff
@@ -10,6 +10,8 @@ export const PATHS: { [key in ROUTES]?: string } = {
   [ROUTES.WORKFLOWS]: 'workflows/notification-workflows',
   [ROUTES.TENANTS]: 'tenants/introduction',
   [ROUTES.TRANSLATIONS]: 'content-creation-design/translations',
+  [ROUTES.STUDIO_ONBOARDING]: 'echo/quickstart',
+  [ROUTES.STUDIO_ONBOARDING_PREVIEW]: 'echo/concepts/workflows',
```

Comment:
**thought:**	should we rename this one to another name so we wont have echo naming here?

Reply:
These are paths in our documentation and should be changed when we have renamed them in there.